CMakeLists.txt

```diff
@@ -106,6 +110,14 @@ if(Omega_h_USE_Kokkos)
   endif()
 endif()
 
+if (Omega_h_USE_KOKKOS_MEM_SPACE STREQUAL "DEVICE")
+  add_definitions(-DOmega_h_MEM_SPACE_DEVICE)
```

Comment:
`Omega_h_MEM_SPACE_DEVICE` should be set/exposed  via `bob_option(...)`. This will automatically get us entries in `Omega_h_config.h` needed for `Omega_h_kokkos.hpp`.

Reply:
This now uses `bob_option(...)`